snooty.toml

```
@@ -9,6 +9,8 @@ sharedinclude_root = "https://raw.githubusercontent.com/10gen/docs-shared/main/"
 [constants]
 package-name-org = "docs-ecosystem"
 stable-api = "Stable API"
+node-driver-long = "MongoDB Node.js driver"
+node-driver-short = "Node.js driver"
```

Really small thing: Because these are technically a product name (as opposed to a driver in the abstract), it should be capital-D.

```suggestion
node-driver-long = "MongoDB Node.js Driver"
node-driver-short = "Node.js Driver"
```